packages/eas-json/schema/eas.schema.json

```diff
@@ -244,10 +252,10 @@
           "description": "Gradle task that will be used to build your project, e.g. `:app:assembleDebug` to build a debug binary.",
           "markdownDescription": "Gradle task that will be used to build your project, e.g. `:app:assembleDebug` to build a debug binary.\n\nIt's not recommended unless you need to run a task that `buildType` does not support, it takes priority over `buildType` and `developmentClient`."
         },
-        "artifactPath": {
+        "applicationArchivePath": {
```

Comment:
I do not have an opinion on that, but this schema is not distributed with eas-cli, so after publishing artfiactPath will be marked as error/unknown property in vscode, even if someone is on older eas-cli that does still support it.

Reply:
I added `artifactPath` back and marked as deprecated.